src/background/paused.js

```diff
@@ -77,8 +77,7 @@ OptionsObserver.addListener('paused', async (paused, lastPaused) => {
     (lastPaused ||
       // Managed mode can update the rules at any time - so we need to update
       // the rules even if the paused state hasn't changed
-      (__PLATFORM__ !== 'firefox' &&
-        (await store.resolve(ManagedConfig)).disableUserControl))
+      (await store.resolve(ManagedConfig)).trustedDomains.length > 0)
```

Comment:
It is not a 100% correct solution, as if the config will update from a list of domains to an empty list, it won't update. The correct solution requires calling `getDynamicRules()` (returns approximately 2k rules) and finding the rule for each SW start.